### PR TITLE
improve manual

### PR DESCRIPTION
**Pull request by jaakdentrekhaak**
improve / correct the markdown files and some minor bugfixes

### CODE REVIEW

File: code/example2/src/controllers/users/handlers/update.handler.ts [hunk 1]
```diff
@@ -3,12 +3,12 @@ import { NextFunction, Request, Response } from 'express';
...
-  res.locals.body = updated;
-  next();
+	const id = Number(req.params.id);
+	const user = UserStore.get(id);
+	if (!user) {
+		return res.status(404).json({ error: "User not found" });
+	}
+	const updated = UserStore.update(id, res.locals.body);
+	res.locals.body = updated;
+	next();
```
Comment (sebastiaanviaene) [inline code/example2/src/controllers/users/handlers/update.handler.ts:13]: no formatting

File: code/example2/src/controllers/users/user.route.ts [hunk 2]
```diff
@@ -11,56 +11,56 @@ import { getList } from './handlers/getList.handler.js';
 import { update } from './handlers/update.handler.js';
 
 const adminMiddleware = (req: Request, res: Response, next: NextFunction) => {
-  if (req.header("auth") !== "api-key") {
-    return res.status(401).send("Unauthorized");
-  }
-  next();
+	if (req.header("x-auth") !== "api-key") {
```
Comment (sebastiaanviaene) [inline code/example2/src/controllers/users/user.route.ts:14]: no formatting

File: code/example2/src/tests/handlers/user.handler.test.ts [hunk 3]
```diff
@@ -72,7 +72,7 @@ describe("Handler tests", () => {
 
 		it("should create user", async () => {
 			let res: User;
-			const body = {
+			const body: Omit<User, "id"> = {
```
Comment (sebastiaanviaene) [inline code/example2/src/tests/handlers/user.handler.test.ts:75]: just use `Partial<User>`

File: code/example2/src/tests/handlers/user.handler.test.ts [hunk 4]
```diff
@@ -84,14 +84,18 @@ describe("Handler tests", () => {
 			expect(res.password).undefined;
 		});
 
-		it("should update user", async () => {
-			const res = {locals: {}} as Response;
-			const body = {
+		it("should update user", () => {
+			const body: Omit<User, "name" | "password" | "id"> = {
```
Comment (sebastiaanviaene) [inline code/example2/src/tests/handlers/user.handler.test.ts:88]: partial user is fine

File: code/example3/src/tests/integration/user.integration.test.ts [hunk 5]
```diff
@@ -4,88 +4,89 @@ import { StatusCode } from '@panenco/papi';
...
-  describe('User Tests', () => {
-    let request: supertest.SuperTest<supertest.Test>;
-    let orm: MikroORM<PostgreSqlDriver>;
-    before(async () => {
-      const app = new App();
-      await app.createConnection();
-      orm = app.orm;
-      request = supertest(app.host);
-    });
+	describe('User Tests', () => {
```
Comment (sebastiaanviaene) [inline code/example3/src/tests/integration/user.integration.test.ts:13]: no formatting

File: content-module2.md [hunk 6]
```diff
@@ -1036,7 +1106,10 @@ it("should CRUD users", async () => {
 
 </details>
 
-### Dependency patching
+# Dev notes
+
+<details>
+<summary>Dependency patching</summary>
```
Comment (sebastiaanviaene) [inline content-module2.md:1112]: move this up as well?

**Review by sebastiaanviaene — CHANGES_REQUESTED** [review]

File: content-module2.md (hunk 6 above)
Comment (jaakdentrekhaak) [reply content-module2.md:1112]: This information is not for the person who has to go through the course. I explained what they needed to do in `Class validator & Class Transformer` > `Intro` > `NOTE`.

**Review by sebastiaanviaene — APPROVED** [review]

**Pull request merged**